.github/workflows/fabric_gen.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Run fabric generator flow
         run: |
           FABulous -c demo
-          FABulous demo --script ./demo/FABulous.tcl
+          FABulous demo --TCLscript ./demo/FABulous.tcl
```

Comment:
```suggestion
          FABulous demo --TCLScript ./demo/FABulous.tcl
```

Reply:
Besides this small typo, everything looks good! 

Since we moved the repo, our checks somehow stopped working, but I need to investigate what the problem is.